pylintrc

```diff
@@ -58,29 +58,289 @@ persistent = yes
 load-plugins = caniusepython3.pylint_checker,edx_lint.pylint
 
 [MESSAGES CONTROL]
+enable = 
```

Comment:
Is this all taken from edx-platform rules?

Reply:
I just re-ran `edx_lint write pylintrc`.  The defaults in edx-lint have changed since the cookiecutter was first created, I think mainly due to a big pylint usage cleanup Cale did.